### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by andrew17752985**
https://andrew17752985.github.io/react_tabs-js/

### CODE REVIEW

File: src/App.jsx
Comment (maxim2310) [inline src/App.jsx:19]: you need to to this condition in tabs component

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just one tiny thing needs to be fixed before approval)

File: src/App.jsx
Comment (TarasHoliuk) [inline src/App.jsx:20]: you should handle this:
image

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
image
try to fix the tests or feel free to ask in the chat

**Review by volodymyr-soltys97 — APPROVED** [review]

**Pull request closed without merging**
